### ISSUE TITLE
NullPointerException while generating PDF/ODT report

### ISSUE THREAD

**Issue by zapbot** [zaproxy/zaproxy#1612]
```
What steps will reproduce the problem?
1. Record the application in ff
2. scan the application
3. in Menu Report--> Generate Report
4. popup will appear
5. Click on "Save"
6. Observe that "Report Generation Failed" message appeared

Find the screenshot for more info

What is the expected output? What do you see instead?
Expected: PDF report , but "Report Generation Failed" appeared

What version of the product are you using? On what operating system?
Windows 8.1 , ZAP 2.4.0

Please provide any additional information below.


```

Original issue reported on code.google.com by `narayana573` on 2015-04-23 13:04:08

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions@ef0d896173e21756fed9b13a3ea299962d4b8de7

**Comment by thc202** [comment]
Version 14 of "Report alert generator" has been released to marketplace which includes this fix.
